pkg/tasks/tasks.go

```diff
@@ -64,24 +66,24 @@ func (tg *TaskGroup) RunConcurrently() (string, error) {
 		g.Go(func() error {
 			klog.V(2).Infof("running task %d of %d: %v", i+1, tgLength, ts.Name)
 			err := ts.Task.Run()
-			klog.V(2).Infof("ran task %d of %d: %v", i+1, tgLength, ts.Name)
 			if err != nil {
-				return taskErr{error: errors.Wrapf(err, "running task %v failed", ts.Name), name: ts.Name}
+				klog.V(2).Infof("encountered error when running task : %v", ts.Name)
```

Comment:
the error would be logged twice (e.g. in `reportError()`), I'd advise to remove this line and continue to log `running task %d of %d: %v`/`ran task %d of %d: %v` before/after `ts.Task.Run()` as in the current version.

Reply:
This log is at a task level, the reportError is a cumulative log

Reply:
I think that too many log messages are confusing ([example](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-monitoring-operator/1238/pull-ci-openshift-cluster-monitoring-operator-master-e2e-agnostic-operator/1417710720845352960/artifacts/e2e-agnostic-operator/gather-extra/artifacts/pods/openshift-monitoring_cluster-monitoring-operator-69d96d879b-htxrb_cluster-monitoring-operator.log)). I agree though that it's probably better to log the error per task and leave the details out at the caller level.

```suggestion
				klog.Warningf("running task %d of %d: %v failed: %v", i+1, tgLength, ts.Name, err)
```